docs/cvl/methods.md

```diff
@@ -346,8 +346,18 @@ determine whether the call should be replaced by an approximation.
 To determine whether a function call is replaced by an approximation, the
 Prover considers the context in which the function is called in addition to the
```

Comment:
writing "together with" instead of "in addition to" would have helped me a good bit, I think 
 --> it's more like that the context (which is the "whether it's resolved status" of the function, or something like this, right??.. EDIT: ah, no .. it's not ...) is a parameter to the policy or so. The "in addition to" makes it feel like they're kind of independent factors. If that makes sense ..

... but I think the paragraph needs something extra anyway -- see below ..